core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -81,6 +82,9 @@ object KafkaServer {
   }
 
   val MIN_INCREMENTAL_FETCH_SESSION_EVICTION_MS: Long = 120000
+  private val EXPONENTIAL_BACKOFF_MULTIPLIER = 2
+  private val EXPONENTIAL_BACKOFF_MAX_INTERVAL_MS = 10 * 60 * 1000  // 10 minutes
+  private val EXPONENTIAL_BACKOFF_JITTER = -0.01 // no randomness
```

Comment:
Krit explicitly requested jitter to avoid thundering herd issues. From the ExponentialBackoff.java implementation, this does indeed seem to disable it...AND it will throw an IllegalArgumentException. I don't understand what you're trying to do with this, but please fix.

Reply:
Yes, I did this on purpose to avoid jittering because I do not think it is necessary. The request rate is already very low, i.e., 1 request per 5 seconds. And controller has a queue to handle concurrent requests. So, I do not think it is necessary to random the retry interval. And it won't throw IllegalArgumentException.

Reply:
Hmm, Java docs seem to disagree:  https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ThreadLocalRandom.html#nextDouble-double-double-

Reply:
Discussed in Slack: there's a code check for negative jitter that avoids the nextDouble() call altogether, so non-issue.